src/components/Button.astro

```diff
@@ -0,0 +1,32 @@
+---
+interface Props {
+    variant?: keyof typeof Variant;
+    size?: keyof typeof Size;
+    icon?: string;
+    disabled?: boolean;
+    label: string;
+    onClick?: () => void;
+}
+
+const style = 'inline-flex items-center justify-center rounded-md border disabled:cursor-not-allowed disabled:bg-primary-100 disabled:text-primary-400';
+
+const Variant = {
```

Comment:
Yay is the time for naming convention 

So, in typescript is common to see that variables that work as absolute constants (hardcoded values) are in snake case

For example it should be

`VARIANTS`

Reply:
So, the names should be in upper case?